tests/foreman/cli/test_remoteexecution.py

```diff
@@ -122,6 +122,8 @@ def test_positive_run_default_job_template_by_ip(self, fixture_vmsetup, fixture_
             and task can be listed by name and ID
 
         :BZ: 1647582
+
+        :parametrized: yes
```

Comment:
Why we are adding `:parametrized: yes` here in all this file. I dont see markup has been used in the test case level

Reply:
All of the tests using the `fixture_vmsetup` fixture are parametrized, because the fixture is parametrized.

Reply:
betelgeuse is supposed to read metadata from module and class before reading function. So if there is some metadata value common for everything in class or file, there is no reason to duplicate it for each individual test.

Reply:
Its not required on all tests.

It has been added to the tests using the `fixture_vmsetup` fixture, which itself is parametrized. Putting it at the module would apply it to test cases that are not parametrized.

Reply:
> It has been added to the tests using the `fixture_vmsetup` fixture

Which are all but one test inside `TestRemoteExecution` class. If that test could be moved outside of class, `parametrized` metadata could be moved to class level.

Although I never considered metadata duplication blocking. That was just FYI.

> Putting it at the module would apply it to test cases that are not parametrized.

That is correct.